src/components/pager/pager.component.js

```diff
@@ -51,36 +100,46 @@ const Pager = (props) => {
     const show = I18n.t('pager.show', { defaultValue: 'Show' });
     const elem = (
       <PagerSizeOptionsInnerStyles>
-        <span>{ show }</span>{ sizeSelector() }<span>{ records(props.pageSize) }</span>
+        <span>{ show }</span>{ sizeSelector() }<span>{ records(currentPageSize) }</span>
       </PagerSizeOptionsInnerStyles>
     );
 
-    return props.showPageSizeSelection ? elem : null;
+    return showPageSizeSelection ? elem : null;
   }
 
   return (
     <PagerContainerStyles data-component='pager'>
       <PagerSizeOptionsStyles>{pageSizeOptions()}</PagerSizeOptionsStyles>
       <PagerNavigation
         { ...props }
-        currentPage={ currentPage }
-        setCurrentPage={ setCurrentPage }
-        setCurrentThemeName={ setCurrentTheme }
+        pageSize={ currentPageSize }
+        currentPage={ page }
+        setCurrentPage={ setPage }
+        totalRecords={ totalRecords || 0 }
```

Comment:
I think that you could just set `totalRecords` to `0` as default

Reply:
fixed 4a893a7